### DISCUSSION TITLE
is check_pgactivity compatible with Icinga2?

### DISCUSSION THREAD

**Discussion by JVC-MSF** [OPMDG/check_pgactivity#316] [Q&A] [answered]
Hi.

I have just a simple question :smiley: 

Is check_pgactivity compatible with Icinga2?

It looks like the tool we are looking for. But our already existing monitoring platform is based in Icinga2.

Thanks!
Javier Vilarroig

**Comment by ioguix** [comment]
Hi,

> Is check_pgactivity compatible with Icinga2?

Yes, it does.

We are currently moving from Naemon to Icinga and various PostgreSQL instances are already monitored from Icinga using check_pgactivity.

**Comment by frost242** [comment, marked as answer]
Hello,
check_pgactivity works flawlessly with Icinga2. We use Icinga with check_pgactivity successfully at my workplace.
Kind regards

**Comment by JVC-MSF** [comment]
Thanks a lot for the answers.

I will plan for including it on our testing set.
